src/siirtc.c

```diff
@@ -378,7 +385,11 @@ bool8 SiiRtcSetAlarm(struct SiiRtcInfo *rtc)
     return TRUE;
 }
 
+#ifndef UBFIX
 static int WriteCommand(u8 value)
+#else
+static int WriteCommand(u8 value)
```

Comment:
Why is there an if else with the same function signature?

Reply:
> Why is there an if else with the same function signature?
I do think this got addressed in another one of their commits